src/styles/stars.scss

```diff
@@ -0,0 +1,17 @@
+.stars {
+  display: flex;
+
+  &__star {
+    width: 16px;
+    height: 16px;
+    border-radius: 0.5px;
+    background-position: center;
+    margin-right: 4px;
+    background-repeat: no-repeat;
+    background-image: url(../images/star.svg);
+
+    &:nth-child(-n + 4) {
+      background-image: url(../images/star-active.svg);
+    }
```

Comment:
This rule will make first 4 stars always active, which is incorrect. You need to apply active stars based on the modifier of the parent element. Consider using a loop to generate classes like '.stars--1', '.stars--2', etc., where the number represents the amount of active stars.